RevenueCatUI/CustomerCenter/Views/RestorePurchasesAlert.swift

```diff
@@ -166,32 +153,27 @@ struct RestorePurchasesAlert: ViewModifier {
                 message += "\n\n" + localization[.updateWarningDescription]
             }
             return message
-        case .restorePurchases:
-            return localization[.goingToCheckPurchases]
         }
     }
 
     private func dismissAlert() {
-        self.alertType = .restorePurchases
         self.isPresented = false
+        Task {
+            await self.customerCenterViewModel.loadScreen()
+        }
```

Comment:
Had to add this @facumenzella @ajpallares since I broke it with the refactor

Not sure it's entirely correct, but the loading needs to go through `customerCenterViewModel` since that's what we are reloading. I think the alternative is to add a `onDismiss()` to the `RestorePurchasesAlert`, add references to `customerCenterViewModel` in `NoSubscriptionsView` and call `self.customerCenterViewModel.loadScreen()` on the callers (`ManageSubscriptionsView` and `NoSubscriptionsView`). 

Wdyt?